src/docs/tutorial-transaction-process/customize-pricing/index.md

```diff
@@ -0,0 +1,469 @@
+
+        const updatedValues = {
+          price,
+          publicData: {
+            cleaningFee: { amount: cleaningFee.amount, currency: cleaningFee.currency },
+          },
+        };
+        onSubmit(updatedValues);
+      }}
+```
+
+### Initialize the form
+
+Next we want to pass inital values for `price` and `cleaningFee`. For this we
+need to get the `cleaningFee` from listing attributes under the `publicData`
+key. Also, because `FieldCurrencyInput` expects the value to be Money object we
+need to convert the value we get from Marketplace API back to instance of Money.
+
+```jsx
+const { price, publicData } = currentListing.attributes;
+const cleaningFee =
+  publicData && publicData.cleaningFee ? publicData.cleaningFee : null;
+const cleaningFeeAsMoney = new Money(cleaningFee.amount, cleaningFee.currency);
+const initialValues = { price, cleaningFeeAsMoney };
+```
+
+### Add input component
+
+We want to be able to save the amount of listing's cleaning fee so we add new
+`FieldCurrencyInput` to the _EditListingPricingForm_. The id and name of this
+field will be `cleaningFee`. Adding this fee will be optional so we don't want
+to add any `validate` param to the `FieldCurrencyInput` like there is in the
+`price` input.
+
+```shell
+└── src
+    └── forms
+        └── EditListingPricingForm
+            └── EditListingPricingForm.js
+```
+
+```jsx
+...
+
+<FieldCurrencyInput
+    id="price"
+    name="price"
+    className={css.priceInput}
+    autoFocus
+    label={pricePerUnitMessage}
+    placeholder={pricePlaceholderMessage}
+    currencyConfig={config.currencyConfig}
+    validate={priceValidators}
+  />
+
+<FieldCurrencyInput
+  id="cleaningFee"
+  name="cleaningFee"
+  className={css.cleaningFeeInput}
+  autoFocus
+  label={cleaningFeeMessage}
+  placeholder={cleaningFeePlaceholderMessage}
+  currencyConfig={config.currencyConfig}
+/>
+...
+```
+
+After adding the new translations keys and some padding to price input in CSS
+file, the EditListingPricingPanel should look something like this:
+
+![EditListingPricePanel](./editlistingpricepanel.png)
+
+# Update BookingDatesForm
+
+In our example the cleaning fee is optional and users can select that as an
+add-on to their booking. In this section, we will add the UI component for
+selecting the cleaning fee and pass the information about the user's choice to
+the FTW-backend.
+
+_Note_: In case you would like to add the cleaning fee automatically to every
+booking, you don't need to add the UI component for selecting the cleaning fee
+and you can move forward to the next section: Add a transaction line item for
+the cleaning fee.
+
+### Prepare props
+
+The cleaning fee is now saved to listing's public data so we can find it under
+the `publicData` key from listing's attributes. Because we decided that adding a
+cleaning fee to listing's information is optional, we need to check if the
+cleaningFee exists in public data or not.
+
```

Comment:
I think the file location would make sense here. I was not sure where this was supposed to be.

Reply:
Good catch! It seems to be missing one paragraph about BookingPanel but I added that now.